photutils/psf/tests/test_utils.py

```diff
@@ -260,3 +264,104 @@ def test_subtract_psf():
             posflux.rename_column(n, n.split('_')[0] + '_fit')
         residuals = subtract_psf(image, psf, posflux)
         assert np.max(np.abs(residuals)) < 0.0052
+
+
+@pytest.mark.skip(reason="Skipping for CI temporarily, passes locally.")
```

Comment:
```suggestion
@pytest.mark.remote_data
```

This should allow the CI tests to pass with remote data.  Locally you can run `tox -e test -- --remote-data=any` to test this.